chain-impl-mockchain/src/ledger/ledger.rs

```diff
@@ -1140,6 +1157,32 @@ impl Ledger {
         Ok(self)
     }
 
+    pub fn apply_encrypted_vote_tally<'a>(
+        mut self,
+        tally: &certificate::EncryptedVoteTally,
+        bad: &TransactionBindingAuthData<'a>,
```

Comment:
Why are these double pointers passed everywhere? The structure is already a thin wrapper around a reference.

Reply:
This is not necessarily to be addressed immediately, just a style remark.